neutron_lbaas/drivers/f5/driver_v2.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import f5lbaasdriver
+
+from neutron_lbaas.drivers import driver_base
+from oslo_log import log as logging
+
+VERSION = "0.1.1"
+LOG = logging.getLogger(__name__)
+
+
+class UndefinedEnvironment(Exception):
+    pass
+
+
+class F5LBaaSV2Driver(driver_base.LoadBalancerBaseDriver):
+
+    def __init__(self, plugin, env='Project'):
+        super(F5LBaaSV2Driver, self).__init__(plugin)
+
+        self.load_balancer = LoadBalancerManager(self)
+        self.listener = ListenerManager(self)
+        self.pool = PoolManager(self)
+        self.member = MemberManager(self)
+        self.health_monitor = HealthMonitorManager(self)
+
+        LOG.debug("F5LBaaSV2Driver: initializing, version=%s, impl=%s, env=%s"
+                  % (VERSION, f5lbaasdriver.__version__, env))
+
+        self.f5 = f5lbaasdriver.v2.bigip.driver_v2.F5DriverV2(plugin, env)
```

Comment:
Paul -- This version is what has been upstreamed to OpenStack, but... it fails with our current version of the driver. Instead, I think we should be using this version: https://github.com/F5Networks/neutron-lbaas/blob/master_pre_upstream/neutron_lbaas/drivers/f5/driver_v2.py. Sorry for the confusion.